src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,125 @@
+
+  // Modifier - active item
+  --pf-c-breadcrumb__item--m-active--Color: var(--pf-c-breadcrumb__item--Color);
+  --pf-c-breadcrumb__item--m-active--hover--Color: var(--pf-c-breadcrumb__item--Color);
+
+  // Modifier - title item
+  --pf-c-breadcrumb__item--m-title--MarginTop: var(--pf-global--spacer--md);
+  --pf-c-breadcrumb__item--m-title--MarginBottom: var(--pf-global--spacer--md);
+  --pf-c-breadcrumb__item--m-title--FontSize: var(--pf-global--FontSize--3xl);
+  --pf-c-breadcrumb__item--m-title--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-breadcrumb__item--m-title--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-breadcrumb__item--m-title--Color: var(--pf-global--Color--300);
+  --pf-c-breadcrumb__item--m-title--hover--Color: var(--pf-global--Color--300);
+
+  display: inline-flex;
+  padding-top: var(--pf-c-breadcrumb--PaddingTop);
+  padding-right: var(--pf-c-breadcrumb--PaddingRight);
+  padding-bottom: var(--pf-c-breadcrumb--PaddingBottom);
+  padding-left: var(--pf-c-breadcrumb--PaddingLeft);
+}
+
+.pf-c-breadcrumb__list {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  list-style: none;
+}
+
+.pf-c-breadcrumb__item {
+  display: flex;
+  align-items: center;
+  font-size: var(--pf-c-breadcrumb__item--FontSize);
+  font-weight: var(--pf-c-breadcrumb__item--FontWeight);
+  line-height: var(--pf-c-breadcrumb__item--LineHeight);
+  color: var(--pf-c-breadcrumb__item--Color);
+  cursor: default;
+
+  &:first-child::before {
+    content: none;
+  }
+
+  &::before {
+    margin-right: var(--pf-c-breadcrumb__divider--MarginRight);
+    margin-left: var(--pf-c-breadcrumb__divider--MarginLeft);
+    font-family: var(--pf-c-breadcrumb__divider--FontFamily);
+    color: var(--pf-c-breadcrumb__divider--Color);
+    content: var(--pf-c-breadcrumb__divider--Content);
+  }
+}
+
+.pf-c-breadcrumb__item.pf-m-active * {
+  color: var(--pf-c-breadcrumb__item--m-active--Color);
+  text-decoration: none;
+  cursor: default;
+
+  &:hover {
+    color: var(--pf-c-breadcrumb__item--m-active--hover--Color);
+    text-decoration: none;
+  }
+}
+
+.pf-c-breadcrumb__item.pf-m-title {
+  flex: 100%;
+  margin-top: var(--pf-c-breadcrumb__item--m-title--MarginTop);
+  margin-bottom: var(--pf-c-breadcrumb__item--m-title--MarginBottom);
+  font-size: var(--pf-c-breadcrumb__item--m-title--FontSize);
+  font-weight: var(--pf-c-breadcrumb__item--m-title--FontWeight);
+  line-height: var(--pf-c-breadcrumb__item--m-title--LineHeight);
+  color: var(--pf-c-breadcrumb__item--m-title--Color);
+  cursor: default;
+
+  &:hover {
+    color: var(--pf-c-breadcrumb__item--m-title--hover--Color);
+    text-decoration: none;
+  }
+
+  &::before {
+    content: none;
+  }
+}
+
+.pf-c-breadcrumb__link {
+  font-size: var(--pf-c-breadcrumb__link--FontSize);
+  font-weight: var(--pf-c-breadcrumb__link--FontWeight);
+  line-height: var(--pf-c-breadcrumb__link--LineHeight);
+  color: var(--pf-c-breadcrumb__link--Color);
+
+  &:hover {
```

Comment:
can you add a `.pf-m-hover` class and group that with this selector? Also should `:focus` and `:active` match these styles?